crates/ark/src/lsp/completions/sources/composite/search_path.rs

```diff
@@ -52,8 +52,8 @@ fn completions_from_search_path(
 ) -> anyhow::Result<Option<Vec<CompletionItem>>> {
     let mut completions = vec![];
 
-    const R_CONTROL_FLOW_KEYWORDS: &[&str] = &[
-        "if", "else", "for", "in", "while", "repeat", "break", "next",
+    const KEYWORD_SOURCE: &[&str] = &[
+        "if", "else", "for", "in", "while", "repeat", "break", "next", "function",
```

Comment:
The other side of transplanting the bare `function` completion to the keyword source.